airbyte-ci/connectors/live-tests/src/live_tests/cli.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+from __future__ import annotations
```

Comment:
What version of Python are we running this on, mainly? Seemingly that is obsolete in 3.10, right?